docs/source/AdministratorGuide/Configuration/ConfReference/Registry/Groups/index.rst

```diff
@@ -44,6 +47,7 @@ This subsection is used to describe DIRAC groups registered in the server.
    -   Properties += FullDelegation
    -   Properties += ProxyManagement
    -   Properties += Operator
+   -   Properties += NotDownloadable
```

Comment:
Where does this come from?

Reply:
At first I thought to add this option as properties and forgot to delete it, now it's fixed